src/analyzer/atsp/load.cpp

```diff
@@ -297,29 +297,22 @@ bool ATSP::loadFromINIFile(const String& filename)
 
 bool ATSP::checkStudyVersion() const
 {
-    auto v = Data::StudyTryToFindTheVersion(pStudyFolder);
-    switch (v)
-    {
-    case Data::versionUnknown:
+    auto v = Data::StudyVersion::tryToFindTheVersion(pStudyFolder);
+    if (v == Data::StudyVersion::unknown())
     {
         logs.error() << "The folder is not a study";
         return false;
     }
-    case Data::versionFutur:
+    if (v > Data::StudyVersion::latest())
     {
         logs.error() << "The format of the study folder requires a more recent version of Antares";
```

Comment:
also it could be "the study require "required" version or downgrade to "<<Data::StudyVersion::latest()